### ISSUE TITLE
cross_entropy * 100

### ISSUE THREAD

**Issue by lttzzlll** [martin-gorner/tensorflow-mnist-tutorial#46]
cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits=Ylogits, labels=Y_)
cross_entropy = tf.reduce_mean(cross_entropy)*100 

in the second line, why there have to * 100? Anyone who can tell me that? Thanks.

**Comment by martin-gorner** [comment]
It's just a scaling factor to put the training and test cross-Entropy on the same graph at comparable scales. It has no impact on training. The only purpose is to make the visuals nicer.

**Comment by lttzzlll** [comment]
Wow. Thanks very much.
